sisu-osgi/sisu-osgi-connect/src/main/java/org/eclipse/sisu/osgi/connect/PlexusModuleConnector.java

```diff
@@ -141,15 +140,14 @@ public synchronized void installRealm(ClassRealm realm, BundleContext bundleCont
 		if (installBundle(bundleContext, realmBundleName, logger) != null) {
 			installed.add(realmBundleName);
 		}
-		boolean isExtensionRealm = !realmExports.artifacts.isEmpty() || !realmExports.artifacts.isEmpty()
-				|| !realmExports.bundleStartMap.isEmpty();
+		boolean isExtensionRealm = !realmExports.artifacts.isEmpty() || !realmExports.bundleStartMap.isEmpty();
```

Comment:
Why was `!realmExports.bundleStartMap.isEmpty()` removed?

Reply:
`!realmExports.artifacts.isEmpty()` was duplicated and therefore redundant in an or statement, because `a || a == a`.

Reply:
Seems the diff has tricked me... I wonder if this is/was really a duplication or it should actually check something else...

Reply:
I asked myself the same question as well, but I think only you can answer that. :)